src/sentry/options/defaults.py

```diff
@@ -724,6 +724,9 @@
 register(
     "store.save-event-highcpu-platforms", type=Sequence, default=[], flags=FLAG_AUTOMATOR_MODIFIABLE
 )
+# The percentage of highcpu traffic we want to route to highcpu dedicated queue. Can remove this
+# option once we fully rollout
+register("store.save-event-highcpu-percent-opt-in", default=0.0, flags=FLAG_AUTOMATOR_MODIFIABLE)
```

Comment:
Why the `opt-in` part at the end of the option name?

Reply:
Let me remove `opt-in` since it's not actual "opt-in" model. It's the way we control for rollout